examples/lstv-based-vad/compute-vad.py

```diff
@@ -0,0 +1,174 @@
+#LTSV features for VAD
+#Used in the DARPA RATS program
+
+from __future__ import division
+from __future__ import print_function
+import numpy as np
+from scipy import signal
+from scipy import version
+import matplotlib.pyplot as plt
+import matplotlib.mlab as mlb
+import ARMA, LTSV, DCTF
+
+import sys,pdb
+
+from kaldi.util.options import ParseOptions
+from kaldi.feat.window import FrameExtractionOptions
+from kaldi.matrix import Vector
+from kaldi.util.table import (MatrixWriter, RandomAccessFloatReaderMapped,
+                              SequentialWaveReader, VectorWriter)
+
+def show_plot(segment_times, sample_freqs, spec, wave, wav_data, vad_feat):
+  """This function plots the vad against the signal and the spectrogram.
+
+  Args:
+      segment_times: the time intervals acting as the x axis
+      sample_freqs: the frequency bins acting as the y axis
+      spec: the spectrogram
+      wave: the wave file in kaldi format (including wave headers)
+      wav_data: the wave data (the audio part of the wave)
+      vad_feat: The calculated vad
+
+  """
+  plt.subplot(3, 1, 1)
+  plt.pcolormesh(segment_times, sample_freqs, 10*np.log10(spec), cmap='jet')
+  plt.ylabel('Frequency [Hz]')
+  plt.xlabel('Time [sec]')
+
+  plt.subplot(3,1,2)
+  axes = plt.gca()
+  axes.set_xlim([0, wave.duration])
+  tmp_axis = np.linspace(0, wave.duration, wav_data.shape[0])
+  plt.plot(tmp_axis, wav_data/np.abs(np.max(wav_data)))
+  plt.xlabel('Time [sec]')
+
+  plt.subplot(3,1,3)
+  axes = plt.gca()
+  axes.set_xlim([0, wave.duration])
+  tmp_axis = np.linspace(0, wave.duration, vad_feat.shape[0])
+  plt.plot(tmp_axis, vad_feat)
+  plt.xlabel('Time [sec]')
+
+  plt.savefig('test', bbox_inches='tight')
+  
+def compute_vad(wav_rspecifier, feats_wspecifier, opts):
+  """This function computes the vad based on ltsv features.
+  The output is written in the file denoted by feats_wspecifier,
+  and if the test_plot flaf is set, it produces a plot.
+
+  Args:
+      wav_rspecifier: The decoder.
+      opts: The symbol table. If provided, "text" output of
+          :meth:`decode` includes symbols instead of integer indices.
+
```

Comment:
Seems like you forgot to update these two argument descriptions after copying them :)